.github/workflows/quicktest.yml

```
@@ -17,28 +17,38 @@ concurrency:
 jobs:
   # Run CI checks/tests which have no dependency on the PHPCS version used.
   quicktest-php:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
+        os: ['ubuntu-latest', 'windows-latest']
         php: ['5.4', 'latest']
 
-    name: "QTest + Lint: PHP ${{ matrix.php }}"
+    name: "QTest + Lint: PHP ${{ matrix.php }} - OS ${{ matrix.os }}"
```

The official image names are long and make the titles less easy to scan. This should improve their readability.

```suggestion
    name: "QTest + Lint: PHP ${{ matrix.php }} (${{ matrix.os == 'windows-latest' && 'Win' || 'Linux' }})"
```

---

That is a fair point. I will update both files once we decide which jobs should run against Windows.